src/main/java/com/karankumar/bookproject/ui/shelf/CustomShelfForm.java

```diff
@@ -141,7 +177,18 @@ private void showSaveError() {
         Notification.show("Could not save shelf.");
     }
 
+    private void showDeleteConfirmation() {
+        Notification.show("Deleted shelf: " + deletingShelf);
+    }
+
+    private void showDeleteError() {
+        Notification.show("Could not delete shelf.");
+    }
```

Comment:
May be we can add a test for this as well?

Reply:
Is there another test around Notifications that I could use as an example? I'm not sure how a test for this should be built.